webui/react/src/services/apiConfig.ts

```diff
@@ -1113,6 +1113,16 @@ export const getActiveTasks: DetApi<
   request: (_, options) => detApi.Tasks.getActiveTasksCount(options),
 };
 
+export const getTaskAllocation: DetApi<
+  Service.GetTaskParams,
```

Comment:
is it always the same param as `GetTask` API? if no, make own param for `getTaskAllocation`